test/js-native-api/test_object/test.js

```diff
@@ -203,29 +203,39 @@ assert.strictEqual(newObject.test_string, 'test string');
   assert.strictEqual(obj.foo, 'baz');
 }
 
+object = Object.create({
+  inherited: 1
+});
+
+object.normal = 2;
+object[Symbol('foo')] = 3;
+Object.defineProperty(object, 'unenumerable', {
+  value: 4,
+  enumerable: false,
+  writable: true,
+  configurable: true
+});
+object[5] = 5;
+
 {
   // Verify that napi_get_property_names gets the right set of property names,
   // i.e.: includes prototypes, only enumerable properties, skips symbols,
   // and includes indices and converts them to strings.
 
-  const object = Object.create({
-    inherited: 1
-  });
-
-  object.normal = 2;
-  object[Symbol('foo')] = 3;
-  Object.defineProperty(object, 'unenumerable', {
-    value: 4,
-    enumerable: false,
-    writable: true,
-    configurable: true
-  });
-  object[5] = 5;
-
   assert.deepStrictEqual(test_object.GetPropertyNames(object),
                          ['5', 'normal', 'inherited']);
 }
 
+{
+  // Verify that napi_get_own_property_names gets
+  // the right set of property names,
+  // i.e.: includes prototypes, only skips symbols,
+  // and includes indices and converts them to strings.
+
+  assert.deepStrictEqual(test_object.GetOwnPropertyNames(object),
+                         ['5', 'normal', 'inherited', 'unenumerable']);
```

Comment:
`inherited` should not be included if this is about *own* property names, right?